internal/resource/config/oauth/client/oauth_client_resource.go

```diff
@@ -0,0 +1,1579 @@
+						Description:         "The JSON Web Encryption [JWE] content encryption algorithm for the ID Token. AES_128_CBC_HMAC_SHA_256 - Composite AES-CBC-128 HMAC-SHA-256, AES_192_CBC_HMAC_SHA_384 - Composite AES-CBC-192 HMAC-SHA-384, AES_256_CBC_HMAC_SHA_512 - Composite AES-CBC-256 HMAC-SHA-512, AES_128_GCM - AES-GCM-128, AES_192_GCM - AES-GCM-192, AES_256_GCM - AES-GCM-256",
+						Optional:            true,
+						Validators: []validator.String{
+							stringvalidator.OneOf("AES_128_CBC_HMAC_SHA_256",
+								"AES_192_CBC_HMAC_SHA_384",
+								"AES_256_CBC_HMAC_SHA_512",
+								"AES_128_GCM",
+								"AES_192_GCM",
+								"AES_256_GCM",
+							),
+							stringvalidator.AlsoRequires(path.MatchRelative().AtParent().AtName("id_token_encryption_algorithm")),
+						},
+					},
+					"policy_group": schema.SingleNestedAttribute{
+						Description: "The Open ID Connect policy. A null value will represent the default policy group.",
+						Optional:    true,
+						Attributes:  resourcelink.ToSchema(),
+					},
+					"grant_access_session_revocation_api": schema.BoolAttribute{
+						Description: "Determines whether this client is allowed to access the Session Revocation API.",
+						Computed:    true,
+						Optional:    true,
+						Default:     booldefault.StaticBool(false),
+					},
+					"grant_access_session_session_management_api": schema.BoolAttribute{
+						Description: "Determines whether this client is allowed to access the Session Management API.",
+						Computed:    true,
+						Optional:    true,
+						Default:     booldefault.StaticBool(false),
+					},
+					"ping_access_logout_capable": schema.BoolAttribute{
+						Description: "Set this value to true if you wish to enable client application logout, and the client is PingAccess, or its logout endpoints follow the PingAccess path convention",
+						Computed:    true,
+						Optional:    true,
+						Default:     booldefault.StaticBool(false),
+					},
+					"logout_uris": schema.SetAttribute{
+						Description: "A list of client logout URI's which will be invoked when a user logs out through one of PingFederate's SLO endpoints.",
+						ElementType: types.StringType,
+						Optional:    true,
+					},
+					"pairwise_identifier_user_type": schema.BoolAttribute{
+						Description: "Determines whether the subject identifier type is pairwise.",
+						Computed:    true,
+						Optional:    true,
+						Default:     booldefault.StaticBool(false),
+					},
+					"sector_identifier_uri": schema.StringAttribute{
+						Description: "The URI references a file with a single JSON array of Redirect URI and JWKS URL values.",
+						Optional:    true,
+						Validators: []validator.String{
+							configvalidators.ValidUrl(),
+						},
+					},
+				},
+				PlanModifiers: []planmodifier.Object{
+					objectplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"client_auth": schema.SingleNestedAttribute{
```

Comment:
Is there a default for this attribute?

Reply:
There's not. It's not always included in the request or returned from the client.

Reply:
default added